src/game/pointers/Pointers.cpp

```diff
@@ -264,8 +263,13 @@ namespace YimMenu
 			NetworkObjectMgr = *ptr.Add(3).Rip().As<void**>();
 		});
 
+		constexpr auto namePtrn = Pattern<"72 ? 78 ? 6E 6E 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 03 00 00 00 00 00 00 F8">("Username");
```

Comment:
Fails to find pattern on my machine.

Reply:
When exactly are you injecting? if you inject at the screen to enter online, or even in the intro r* gun thing it works.

Reply:
It crashed on both the intro and in game. Should work anytime tho no?

Reply:
> It crashed on both the intro and in game. Should work anytime tho no?

It should and now i am very confused because it worked flawlessly on my pc 💀

Reply:
have you gone into the online mode before?

Reply:
Having to inject while playing online is shit, idk how is the ac in this game but that sounds like a terrible idea in any case

Reply:
> Having to inject while playing online is shit, idk how is the ac in this game but that sounds like a terrible idea in any case

i am not saying inject in online, there is a screen where you select the mode you want to go in to. i have had success injecting here.

Reply:
> It crashed on both the intro and in game. Should work anytime tho no?

Just to confirm you went into the online mode right?

Reply:
Yes

Reply:
> Yes

There are no xrefs to the name char array that worked for me 😭

Reply:
> I cant find the pattern either. Have you checked the sig I posted in the signature research discussion? You can spoof your username there (much like GTA used to work back in the day).

The send info to host thingy?

Reply:
I found what he was talking about. I will work on getting it working.
heres the sig if you are wondering:
`48 8B C4 48 89 58 ? 48 89 68 ? 56 57 41 54 41 56 41 57 48 83 EC ? 4D 8B F1 48 8B F9 48 81 C1 ? ? ? ? 4C 8D 48 ? 41 8B E8 4C 8B FA 33 DB E8 ? ? ? ? 84 C0 0F 84 ? ? ? ? 8B B4 24 ? ? ? ? 4C 8D 8C 24 ? ? ? ? 44 8B C5 48 8B CF 44 2B C6 49 8D 14 37 E8 ? ? ? ? 84 C0 0F 84 ? ? ? ? 44 8B A4 24 ? ? ? ? 48 8D 8F ? ? ? ? 44 8B C5 4C 8D 8C 24 ? ? ? ? 45 2B C4 44 2B C6 41 8D 14 34 49 03 D7 E8 ? ? ? ? 84 C0 74 ? 8B 9C 24 ? ? ? ? 48 8D 4C 24 ? 41 03 DC 03 DE E8 ? ? ? ? 8B D3 48 8D 4C 24 ? 2B EB 49 03 D7 44 8B C5 E8 ? ? ? ? 48 8D 97 ? ? ? ? 41 B8 ? ? ? ? 48 8D 4C 24 ? E8 ? ? ? ? 84 C0 74 ? 8B 97`

Reply:
@tyackman do you need to add or rip anything to get the username?

Reply:
> @DayibBaba Yes, it's the same function. @Rxann check old yimmenu versions to find the args but the first should be type netPlayer I believe. That signature is pretty long, I would find somewhere else to hook it so scanning for it doesn't take a year.

How far back are we talking? I am not super familiar with github, so I don't exactly know how to find a previous version. I will just look for prs that look like they change the way names are spoofed.